src/index.html

```diff
@@ -6,13 +6,71 @@
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <title>Product cards</title>
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <img
+        class="card__img"
+        src="./images/imac.jpeg"
+        alt="Product"
+      />
+
+      <section class="card__info">
+        <div class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </div>
+
+        <span class="card__code">Product code: 195434</span>
+
+        <div class="card__reviews">
+          <div class="stars stars--rating-4">
+            <div class="stars__star stars__star--active"></div>
+
+            <div class="stars__star stars__star--active"></div>
+
+            <div class="stars__star stars__star--active"></div>
+
+            <div class="stars__star stars__star--active"></div>
+
+            <div class="stars__star"></div>
+          </div>
+          <div class="card__reviews-value">Reviews: 5</div>
+        </div>
+
+        <div class="card__price">
+          <span class="card__price-title">Price:</span>
+
+          <span class="card__price-value">$2,199</span>
+        </div>
+
+        <a
```

Comment:
Buttons that perform actions but do not navigate to a new page should use the `<button>` tag instead of `<a>`. If this is supposed to be a link that leads to a purchase page, then it is fine to use `<a>`, but make sure to provide a valid `href` attribute.